components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -526,10 +526,14 @@ protected void buildClaimMappings(Map<ClaimMapping, String> claims, Map.Entry<St
 
             }
         } catch (Exception e) {
-            claimValue = new StringBuilder(entry.getValue().toString());
+            if (log.isTraceEnabled()) {
+                log.trace("Error while mapping " + entry.getKey() + " with value " + entry.getValue(), e);
+            }
```

Comment:
@deki, Can we make above a debug log instead of a trace log

Reply:
Sure before there was no logging at all that's why I thought I start with the lowest level. I will adapt that.